dispatcher/config_test.ini

```diff
@@ -0,0 +1,60 @@
+[DEFAULT]
+
+# Poll frequency in seconds for main program loop. It makes
+# some sense to make other time-based options multiples of
+# this, though not required
+PollFrequency = 3
+
+# How long to wait for boards to arm
+ArmTimeout = 40
+
+# How long since a client's last check-in until we consider
+# it to be 'timing out'
+ClientTimeout = 15
+
+# Database URIs. Please store DB password in the MONGO_PASSWORD
+# environment variable and runs DB password as RUNS_MONGO_PASSWORD
+ControlDatabaseURI = mongodb://daq:%%s@gw:27020/admin
+ControlDatabaseName = daq
+RunsDatabaseURI = mongodb://daq:%%s@gw:27017/admin
+RunsDatabaseName = xenonnt
```

Comment:
This is also the production database, it should also be `test`. Also the URI above should be on port 27020, not 27017